frontend/src/components/filters.svelte

```diff
@@ -1,13 +1,30 @@
 <script lang="ts">
   import { filters } from "../stores/filters.js"
+  import { sorting } from "../stores/sorting.js"
   import { currentProviders } from "../stores/providers.js"
 
+  const contentTypes = ["Filters", "Sorting"]
+
   export let search: () => void
+
+  let currentTab = 0
   let isDropdownOpen = false
   let providerLabelText: string
+  let descLabelText: string
+  let ascLabelText: string
 
   $: filterAmount = Object.values($filters).filter(v => v === false).length
 
+  $: {
+    if ($sorting.byPopularity) {
+      descLabelText = "Highest"
+      ascLabelText = "Lowest"
+    } else {
+      descLabelText = "Newest"
+      ascLabelText = "Oldest"
+    }
+  }
+
```

Comment:
Can we make it not use the popularity sorting when the user has input(unless it was actively selected)